components/landing/SearchLanding.vue

```diff
@@ -0,0 +1,133 @@
+<template>
+  <section
+    class="landing-search is-flex is-flex-direction-column is-justify-content-center is-align-items-center">
+    <div class="title">
+      <span>Discover,</span> Collect <span> And Sell</span>
+    </div>
+    <img class="sub-title" :src="landingImage[2]" />
```

Comment:
I would use text instead of image as I feel this text will change in near futur (moon/astar nfts)

Reply:
If using text, we could hardly to achieve the font style as on the Figma design.
<img width="353" alt="image" src="https://user-images.githubusercontent.com/31397967/191050022-22901a22-8de2-4b8a-9170-e08d54599283.png">

> in near futur (moon/astar nfts)

In this case, how about using different image to show?  The number of images is limited.

Reply:
Bruh, this can be done with css
![Capture d’écran 2022-09-19 à 5 15 08 PM](https://user-images.githubusercontent.com/9987732/191052050-04e33243-e185-47c9-a553-356912c5c641.png)

Reply:
Thanks. I will do this.